recipes/pixell/meta.yaml

```diff
@@ -0,0 +1,98 @@
+{% set name = "pixell" %}
+{% set version = "0.26.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pixell-{{ version }}.tar.gz
+  sha256: a84070dfdf3701563f9fc3ea1ec454d7b71351edddb62ff51df1180f05d72ec2
+
+build:
+  skip: true  # [py<39 or win]
+  entry_points:
+    - benchmark-pixell-runner = pixell.scripts.benchmark_pixell_runner:main
+    - benchmark-pixell = pixell.scripts.benchmark_pixell:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib("c") }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - make
+    - cython
```

Comment:
This should only be in `host`
```suggestion
```

Reply:
The build fails with:
```
  ../meson.build:3:0: ERROR: Cython compiler 'cython' cannot compile programs
```

when it is not included.